src/Component/MonsterPlayground.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import 'Stylesheet/MonsterAttack.css'
+import PlayersActionsContainer from 'Component/PlayersActionsContainer'
+function MonsterPlayground() {
+  return (
+    <div className='MonsterDiv'>
```

Comment:
general feedback for all files
- class names should be in the format - "monster-div"

Reply:
Done